contribs/gmf/src/directives/partials/gmflayertree.html

```diff
@@ -0,0 +1,19 @@
+<span ng-if="::!layertreeCtrl.isRoot"
+    ng-model="::gmfLayertreeCtrl.getStyleGetterSetter(layertreeCtrl)"
+    ng-model-options="{getterSetter: true}">
+  <img ng-src="{{::layertreeCtrl.node.icon}}"></img>
+  <input type="checkbox"
```

Comment:
Are we sure we want checkboxes ????
In Luxembourg we use icons from font-awesome.

Ping @pgiraud for that.
If not, we could get completly rid of the checkboxGetterSetter as well.